MHVLib/Methods/MHVMethod.h

```diff
@@ -38,6 +46,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign) int version;
 
 /**
+ A MHVMethodType indicating the method type
+ */
+@property (nonatomic, assign, readonly) MHVMethodType type;
+
+/**
```

Comment:
I debated whether to do this as a type or subclasses of MHVMethod: MHVXMLMethod, MHVRestMethod, MHVDataMethod, etc.  I may still do that...